packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -322,55 +325,51 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
     <Button icon={<CalendarIcon />} title={props.openCalendarTitle} iconOnly disabled={props.disabled} />
   );
 
-  const element = (
-    <Ref innerRef={datepickerRef}>
-      {getA11yProps.unstable_wrapWithFocusZone(
-        <ElementType
-          {...getA11yProps('root', {
-            className: classes.root,
-            ...unhandledProps,
-          })}
-        >
-          {!props.buttonOnly &&
-            createShorthand(Input, input, {
-              defaultProps: () =>
-                getA11yProps('input', {
-                  placeholder: props.inputPlaceholder,
-                  disabled: props.disabled,
-                  error: !!error,
-                  value: formattedDate,
-                  readOnly: !allowManualInput,
-                  required: props.required,
-                  ref: inputRef,
-                  'aria-label': formatRestrictedInput(restrictedDatesOptions, dateFormatting),
-                }),
-              overrideProps: overrideInputProps,
-            })}
-          {createShorthand(Popup, popup, {
-            defaultProps: () => ({
-              open: openState && !props.disabled,
-              content: calendarElement,
-              trapFocus: {
-                disableFirstFocus: true,
-              },
-              trigger: triggerButtonElement,
-              target: !props.buttonOnly ? inputRef.current : null,
-              position: 'below' as const,
-              align: 'start' as const,
+  const element = getA11yProps.unstable_wrapWithFocusZone(
+    <ElementType
+      {...getA11yProps('root', {
+        className: classes.root,
+        ...unhandledProps,
+      })}
+    >
+      {!props.buttonOnly &&
+        createShorthand(Input, input, {
+          defaultProps: () =>
+            getA11yProps('input', {
+              placeholder: props.inputPlaceholder,
+              disabled: props.disabled,
+              error: !!error,
+              value: formattedDate,
+              readOnly: !allowManualInput,
+              required: props.required,
+              ref: inputRef,
+              'aria-label': formatRestrictedInput(restrictedDatesOptions, dateFormatting),
             }),
-            overrideProps: (predefinedProps: PopupProps): PopupProps => ({
-              onOpenChange: (e, { open }) => {
-                // In case the event is a click on input, we ignore such events as it should be directly handled by input.
-                if (!(e.type === 'click' && e.target === inputRef?.current)) {
-                  setOpenState(open);
-                  _.invoke(predefinedProps, 'onOpenChange', e, { open });
-                }
-              },
-            }),
-          })}
-        </ElementType>,
-      )}
-    </Ref>
+          overrideProps: overrideInputProps,
+        })}
+      {createShorthand(Popup, popup, {
+        defaultProps: () => ({
+          open: openState && !props.disabled,
+          content: calendarElement,
```

Comment:
I expect that `trigger`, `target` & `content` will be in `overrideProps`, what should happen if I will specify `<Datepicker popup={{ content: 'foo' }} />`?

Reply:
Good point. Will also change position and align. 

I was though wondering. Does it make sense to override the 'content'? This is the actual calendar that can already be overriden through the calendar slot.

Reply:
> Good point. Will also change position and align.

I think that `position` & `align` should stay in default to be overridable to users.

Reply:
Updated